hack/build-rpm-release.sh

```diff
@@ -57,7 +57,8 @@ make clean
 
 # migrate the tito artifacts to the Origin directory
 mkdir -p "${OS_OUTPUT}"
-mv "${tito_output_directory}"/* "${OS_OUTPUT}"
+cp -R "${tito_output_directory}"/* "${OS_OUTPUT}"
```

Comment:
We originally chose not to do this because it doubles an already huge storage requirement. Any way we can get symlink moving without that overhead?